draft-ietf-cats-framework.md

```diff
@@ -560,6 +560,16 @@ More importantly, the means for identifying a flow for the purpose of ensuring i
 
 This document does not define any mechanism for defining or enforcing service contact instance affinity.
 
+## Policy Delivery
+
+As shown in Figure 1, the control plane and management plane need to communicate some policies to the data plane. The control plane and management plane can be implemented in a control center entity.
```

Comment:
```suggestion
As shown in {{fig-cats-fw}}, the control and management planes need to communicate some policies to the data plane.
```

Reply:
No need to repeat that this can be distributed/centralized as this was already stated in the doc.